src/dialogxml/widgets/scrollbar.hpp

```diff
@@ -45,7 +47,20 @@ class cScrollbar : public cControl {
 	bool vert = true;
 	static std::string scroll_textures[NUM_STYLES];
 	static const rectangle up_rect[NUM_STYLES][4], down_rect[NUM_STYLES][4], bar_rect[NUM_STYLES][4], thumb_rect[NUM_STYLES][4];
+	// Mouse wheel scrolling events inside this rectangle will be handled by the scrollbar.
+	// Should at least cover the scrollbar itself, but can extend outside (example: scrolling
+	// in the inventory area). 
+	rectangle mousewheel_event_catching_rect {0, 0, 0, 0};
```

Comment:
As with the function to set it, this name is too long. (I think explicitly initializing it also isn't necessary, but it can't hurt either, I guess.)

Also as I mentioned elsewhere, there's not really any need to force it to cover the scrollbar itself.

Reply:
done